cmake/BareosCpmPackages.cmake

```diff
@@ -32,10 +32,45 @@ if(FETCHCONTENT_FULLY_DISCONNECTED AND NOT CPM_LOCAL_PACKAGES_ONLY)
 endif()
 include(CPM)
 
-CPMAddPackage(
+# Keep module alphabetically ordered
+
+CPMFindPackage(
+  NAME CLI11
+  VERSION 2.4.2
+  GITHUB_REPOSITORY CLIUtils/CLI11
+  EXCLUDE_FROM_ALL YES
+)
+
+CPMFindPackage(
+  NAME CPMLicenses.cmake 
+  GITHUB_REPOSITORY cpm-cmake/CPMLicenses.cmake
+  VERSION 0.0.5
+)
+
+CPMFindPackage(
   NAME fmt
   VERSION 6.2.1
   GITHUB_REPOSITORY fmtlib/fmt
   GIT_TAG 10.2.1
   EXCLUDE_FROM_ALL YES
 )
+
+CPMFindPackage(
+  NAME xxHash
+  VERSION 0.8.2
+  GITHUB_REPOSITORY Cyan4973/xxHash
+  EXCLUDE_FROM_ALL YES
+  SOURCE_SUBDIR cmake_unofficial
+)
+set(XXHASH_ENABLE_DISPATCH
+  OFF
+  CACHE INTERNAL ""
+)
```

Comment:
that's really bad as it will break xxHash performance.

Reply:
So we have an additional question here: before we build xxHash as static lib, now do we don't just link to it ?
So the performance is not related to us, but the people build libxxhash ?